pipelines/pipeline_util/time_period.py

```diff
@@ -13,3 +14,40 @@ def previous_month():
     start_date = end_date.replace(day=1)
 
     return DateRange(start_date=start_date, end_date=end_date)
+
+
+def get_date_range_phrase(time_period):
+    generator = StartEndDatePhraseGenerator()
+    return generator.get_date_range_phrase(time_period.start_date, time_period.end_date)
```

Comment:
Really minor point, but is there any reason that this method couldn't take a time period instead of separate start and end dates?

Reply:
Nope! I just haven't done it (yet). I reused several utility classes - including this one - from a verify project so that's why the coding style is a bit inconsistent.